Formula/cli53.rb

```diff
@@ -0,0 +1,22 @@
+class Cli53 < Formula
+  desc "command-line tool for Amazon Route 53"
+  homepage "https://github.com/barnybug/cli53"
+  url "https://github.com/barnybug/cli53/archive/0.7.4.tar.gz"
+  sha256 "3cb89e6aa91676ffd0577798a4b06b056667d18ad836de4fa31c0564ee48474e"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/barnybug"
+    ln_s buildpath, buildpath/"src/github.com/barnybug/cli53"
+
+    system "make", "build"
+
+    bin.install "cli53"
+  end
+
+  test do
+    system "#{bin}/cli53", "--version"
```

Comment:
all cli53 need a AWS id and secret to run. Also, all make a request to amazon and a HTTP mock maybe is too much.

Maybe check that running `cli53` without args prints something expected, like `cli53 - manage route53 DNS`

Any thoughts?

```
NAME:
   cli53 - manage route53 DNS

USAGE:
   cli53 [global options] command [command options] [arguments...]

COMMANDS:
   list, l	list domains
   create	create a domain
   delete	delete a domain
   import	import a bind zone file
   export	export a bind zone file (to stdout)
   rrcreate, rc	create one or more records
   rrdelete, rd	delete a record
   rrpurge	delete all the records (danger!)
   dslist	list reusable delegation sets
   dscreate	create a reusable delegation set
   dsdelete	delete a reusable delegation set
   help, h	Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --help, -h		show help
   --version, -v	print the version
```